pandas/plotting/_core.py

```diff
@@ -2852,22 +2852,48 @@ def pie(self, y=None, **kwds):
 
     def scatter(self, x, y, s=None, c=None, **kwds):
         """
-        Scatter plot
+        A scatter plot with point size *s* and color *c*.
+
+        The coordinates of each point *x,y* are defined by two dataframe
+        columns and filled circles are used to represent each point.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
+        x : column name or column position
+            Horizontal coordinates of each point.
+        y : column name or column position
+            Vertical coordinates of each point.
         s : scalar or array_like, optional
             Size of each point.
-        c : label or position, optional
+        c : label, column name or column position, optional
             Color of each point.
-        `**kwds` : optional
+        kwds : optional
             Keyword arguments to pass on to :py:meth:`pandas.DataFrame.plot`.
 
         Returns
         -------
         axes : matplotlib.AxesSubplot or np.array of them
+
+        See Also
+        --------
+        matplotlib.pyplot.scatter : scatter plot using multiple input data
+            formats.
+
+        Examples
+        --------
+
+        .. plot::
+            :context: close-figs
+
+            >>> from sklearn.datasets import load_iris
```

Comment:
We can't import sklearn here, since it isn't installed.

Reply:
Maybe just make a small sample dataset instead.

Reply:
It passed all checks and test and the plot was shown in the docs. But ok, I will write down the dataframe explicitely.